test/screenshot/floating-label/index.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import FloatingLabel from '../../../packages/floating-label';
+
+import '../../../packages/floating-label/index.scss';
+import './index.scss';
+
+class Input extends React.Component {
+  state = {
+    shouldUsePrimaryLabel: true,
+    float: false,
+    width: 0,
+  };
```

Comment:
You shouldnt have all this logic in your screenshot tests. Just render two components...then take a screenshot. No state necessary.